File: repos_estudiantes_ALVAREZ_tercera_entrega/AJATA_CANAVIRI_NICOL_MAYRA/Generecidad/Ejercicio1/Java/Caja.java
package Generecidad.Ejercicio1.Java;

public class Caja<T> { 
    private T contenido; 

    // a)
    public void guardar(T objeto) {
        this.contenido = objeto;
    }

    public T obtener() {
        return contenido;
    }

    @Override
    public String toString() {
        return "Contenido de la Caja: " + contenido;
    }
}

File: repos_estudiantes_ALVAREZ_tercera_entrega/AJATA_CANAVIRI_NICOL_MAYRA/Generecidad/Ejercicio1/python/Caja.py
from typing import TypeVar, Generic

T = TypeVar('T')

class Caja(Generic[T]):
   
    def __init__(self):
        self._contenido: T = None 

    def guardar(self, objeto: T):
        
        self._contenido = objeto

    def obtener(self) -> T:
        
        return self._contenido

    def __str__(self):
        
        return f"Contenido de la Caja: {self._contenido}"